src/components/image/image.tsx

```diff
@@ -39,13 +41,12 @@ export default function Image(props: ImageProps) {
         onLoad={() => {
           setIsLoading(false)
         }}
+        className={!props.width && !props.height ? 'rustic-full-width' : ''}
```

Comment:
why set full width to 100% ? I think we should instead set the [object-fit property](https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit).